projects/igniteui-angular/src/lib/data-operations/filtering-condition.ts

```diff
@@ -568,3 +568,21 @@ export interface IDateParts {
     seconds: number;
     milliseconds: number;
 }
+
+export class InFilteringOperation implements IFilteringOperation {
```

Comment:
Please add documentation for these two classes. If they are not meant to be used by the users add `@hidden`.

Reply:
Done.